app/models/manageiq/providers/ibm_power_vc/inventory/parser/cloud_manager.rb

```diff
@@ -1,2 +1,210 @@
+
+    persister.vms_and_templates_advanced_settings.build(
+      :resource     => server,
+      :display_name => _('Storage Connectivity Group'),
+      :name         => 'storage_connectivity_group',
+      :description  => _('Storage Connectivity Group ID.'),
+      :value        => flavor.metadata["powervm:storage_connectivity_group"],
+      :read_only    => true
+    )
+
+    persister.vms_and_templates_advanced_settings.build(
+      :resource     => server,
+      :display_name => _('ame_expansion_factor'),
+      :name         => 'AME Expansion Factor',
+      :description  => _('Active Memory™ Expansion (AME) factor or specifies that AME is disabled on the virtual machine.'),
+      :value        => get_ame_factor(flavor.metadata["powervm:ame_expansion_factor"]),
+      :read_only    => true
+    )
+
+    persister.vms_and_templates_advanced_settings.build(
+      :resource     => server,
+      :display_name => _('secure_boot'),
+      :name         => 'secure_boot',
+      :description  => _('Secure boot is supported on POWER9 systems running AIX 7.2 TL3 or later.'),
+      :value        => get_secure_boot(flavor.metadata["secure_boot"]),
+      :read_only    => true
+    )
+
+    persister.vms_and_templates_advanced_settings.build(
+      :resource     => server,
+      :display_name => _('Simplified Remote Restart Capability'),
+      :name         => 'srr_capability',
+      :description  => _('Ability to remote restart the LPAR to supported CEC or host when the source CEC or host is down.'),
+      :value        => flavor.metadata["srr_capability"] || "false",
+      :read_only    => true
+    )
+
+    persister.vms_and_templates_advanced_settings.build(
+      :resource     => server,
+      :display_name => _('shared_proc_pool_name'),
+      :name         => 'shared_proc_pool_name',
+      :description  => _('Specifies the shared processor pool to be targeted during deployment of a virtual machine.'),
+      :value        => flavor.metadata["powervm:shared_proc_pool_name"],
+      :read_only    => true
+    )
+
+    persister.vms_and_templates_advanced_settings.build(
+      :resource     => server,
+      :display_name => _('Enable LPAR metric'),
+      :name         => 'enable_lpar_metric',
+      :description  => _('Allows performance information collection for the virtual machine.'),
+      :value        => flavor.metadata["powervm:enable_lpar_metric"],
+      :read_only    => true
+    )
+  end
+
+  def get_ame_factor(ame_input)
+    ame_factor = ame_input
+    ame_factor = "disabled" if !ame_input || ame_input == "0"
+    ame_factor
+  end
+
+  def get_secure_boot(secure_boot_input)
+    case secure_boot_input
+    when "0"
+      "disabled"
+    when "1"
+      "log only"
+    when "2"
+      "enforce"
+    else
+      ""
+    end
+  end
+
+  def calculate_pinning(vm)
+    pin_vm = "false"
+    move_pin_vm = "false"
+    vm.metadata.each do |item|
+      case item.key
+      when "pin_vm"
+        pin_vm = item.value
```

Comment:
I think the code below would look cleaner if `pin_vm` and `move_pin_vm` were booleans, e.g. `pin_vm = item.value == "true"` then you can do e.g. `pinning = "hard" if pin_vm && !move_pin_vm`